### ISSUE TITLE
ramdisk update failure causes issues with not being able to implement bootui support

### ISSUE THREAD

**Issue by wazeeahmed** [chenxiaolong/DualBootPatcher#200]
As referred in issue #176 ; the ramdisk update option is faulty . Now I can't even update the ramdisk for adding new bootui option recently added . Any light on that ?

**Comment by hankching** [comment]
Pls look /internal sd/multiboot/(each slot)/boot.img that exists or not.

**Comment by wazeeahmed** [comment]
yes it does . But problem is my device doesn't have internal sd . twrp recovery by 1cefire has support for both sprout4 & 8 support . My device is 8gb ; but it's sprout4 by default . The recovery created the data/media/0 folder . But dbp always looks inside the /storage/34AA-2FD7/multiboot instead of /data/media/0/multiboot . Thats what causes the issue .

**Comment by hankching** [comment]
Ooh, I can't help.

**Comment by wazeeahmed** [comment]
@chenxiaolong Hope you have a fix for this . It's really bugging me for days .

**Comment by chenxiaolong** [comment]
This will not be fixed in time for 9.0.0 because it's more complex than I originally anticipated. A workaround for the ramdisk update is to patch and reflash your kernel or ROM (they both result in the same thing).

**Comment by wazeeahmed** [comment]
But my device kernels only contain zimage and screipts for bootimage extraction . The bootimage gets extracted when flashing a kernel and gets the zimage replaced . Can't there be options like selecting internal sd or external sd for seting whatever our choice is ? That way rom switch files won't have issues and neither does Ramdisk update will have .
